generator/generator_tests/restriction_test.cpp

```diff
@@ -95,7 +95,7 @@ void TestRestrictionBuilding(string const & restrictionContent, string const & m
 
   // Adding restriction section to mwm.
   string const restrictionFullPath = my::JoinPath(writableDir, restrictionRelativePath);
-  string const mwmFullPath = my::JoinPath(writableDir, mwmRelativePath);
+  string const & mwmFullPath = scopedMwm.GetFullPath();
```

Comment:
Please do such minor changes in separate PRs in the future. This one is big enough already.

Reply:
okey